addon/-private/internals.js

```diff
@@ -1,5 +1,11 @@
 import Ember from 'ember';
 
-const ClosureActionModule = Ember.__loader.require('ember-routing-htmlbars/keywords/closure-action');
+let ClosureActionModule;
+
+try {
```

Comment:
It would be better to just check if the module is present, instead of try/catch. 

Ember.__loader.registry or something like that